### DISCUSSION TITLE
Allow you to transfer ownership of a project

### DISCUSSION THREAD

**Discussion by cam-duncan** [community/community#11343] [Projects and Issues]
I created a personal project and eventually wanted to move it to an organization. I was able to move all of my repos but then I had to _manually_ re-enter all of the issues. Without a way to transfer ownership and no multi-select/export, it was not a pleasant experience 😂.

**Comment by swiftgame** [comment]
I am also on the verge of doing such a transfer, and I really would like to know if this is a known issue. If it is, then I am not going to go through with the tranfser. Thanks for posting.

**Reply by cam-duncan** [reply]
@swiftgame I was not able to find a way to transfer at least. If I had to do it again, though, I would make sure to convert all of my issues from draft to an actual issue before transferring the repos. Then when you create a new project you can fairly easily add existing issues without as much copy and paste.

**Comment by DrOteonu-Nanquanu** [comment]
Ah, thanks for the advice. Moreover I found that it seems that the problem of meta-data-loss only occurs if you have a free plan:

After accepting this repository transfer, you will lose access to:
  - Codeowners functionality.
  - Any existing wikis.
  - Pulse, Contributors, Community, Traffic, Commits, Code Frequency and Network on the Insights page.
  - Draft PRs.
  - Multiple assignees for issues and PRs.
  - Multiple PR reviewers.
  - Branch protection rules.
  - Any existing Pages.

You can upgrade your plan before accepting the transfer of this repository to avoid losing access.

**Comment by UchiTesting** [comment]
Hello,
I am also facing this issue. I see there is no way out.
I will recreate my project. 
That will be a quite some content to copy/paste.
Fortunately I started my project yesterday.
Hopefully this feature will be added later on.
I also tried to invite the organisation to the project in an attempt to at least have access to it.
It did not work. Only user accounts seem to be eligible for invite.
Regards.

Edit:
Actually this is easier than I thought.
I thought it is weird I cannot access the project as issues belong to repo and it was transferred to the organization.
You can simply add the items with `#` (I did not pay attention to that)
Then pick your project and the issues list appear.
It then opens that list automatically so it is just a matter of clicking.
Moreover it displays only what is not already there.
Took me up to 2 minutes and rather felt like 30s.

![image](https://user-images.githubusercontent.com/56003633/187433049-c6e1f6d0-987a-4398-8db8-55e5db0e00ab.png)

**Reply by AnilGupte** [reply]
I have the same message and problem.  Will I lose access to activity especially Forks and Pull requests?

One important question is: Was this available to the free plan before?  Did this happen after Microsoft's takeover?
